app/react/Library/helpers/requestState.js

```diff
@@ -39,10 +39,13 @@ export function processQuery(params, globalResources, key) {
   const { userSelectedSorting, ...sanitizedQuery } = query;
 
   const loggedIn = globalResources.user && globalResources.user.has('role');
+  const isAdmin = loggedIn && globalResources.user.get('role') === 'admin';
```

Comment:
Do we usually do these kind's of checks by string?

Reply:
Good point, done.